src/structures/Message.js

```diff
@@ -214,8 +241,11 @@ class Message extends Base {
       this,
       'mentions' in data ? data.mentions : this.mentions.users,
       'mentions_roles' in data ? data.mentions_roles : this.mentions.roles,
-      'mention_everyone' in data ? data.mention_everyone : this.mentions.everyone
+      'mention_everyone' in data ? data.mention_everyone : this.mentions.everyone,
+      'mention_channels' in data ? data.mention_channels : this.mentions.crosspostedChannels
     );
+
+    this.flags = new MessageFlags('flags' in data ? data.flags : this.flags.bitfield).freeze();
```

Comment:
It looks like Discord does not send `flags` at all if they are 0.
This leads to `SUPPRESS_EMBEDS` not getting cleared if you unsupress a message.
(Although that endpoint is at the moment technically not """officially supported""" by Discord)

Reply:
Would it be safe to assume that if `'flags' in data` isn't true, we don't have a message field? How do we even test that? Can you suppress the embeds of a crossposted message? If so, I'll change that 👍

Reply:
> Would it be safe to assume that if `'flags' in data` isn't true, we don't have a message field?

What `message` field do you mean?

From what I've seen during a bit of testing it seems like the field is missing if no flags are present.
I haven't seen any cases where there were any flags but the property missing.
>Can you suppress the embeds of a crossposted message?  

Yes, you can. Both flags are present then, unsuppressing them later sends another update with just the crosspost flag. (Behavior is as you'd expect it)

Reply:
> What `message` field do you mean?
I meant message flags. Close enough 😅 


> Yes, you can. Both flags are present then, unsuppressing them later sends another update with just the crosspost flag. (Behavior is as you'd expect it)

Perfect! Then, we default to 0 if 'flags' isn't in data when we patch?